app/assets/stylesheets/pages/_show.scss

```diff
@@ -65,7 +65,11 @@ p {
 
   }
 
-
+img {
```

Comment:
I don't think you want to do that, it will apply to ALL images on the website. You should use a more specific selector than `img`, what about using a class?